xml/networking-ts_dpdk.xml

```diff
@@ -176,8 +176,6 @@ NUMA node1 CPU(s):     12-23,36-47</screen>
     </para>
    </listitem>
    <listitem>
-    <!-- SLES 12 SP3 has OVS 2.7 already. Do we still need the following? -
-    sknorr, 2018-07-05 -->
```

Comment:
Very not in scope of this PR, but has anyone had a chance to look at this comment @csymons-suse or @dmpop ?

Reply:
No one complained about this since 2018-07-05, so I think it can go.